File: DS Assignments/ProgrammingAssignment1/ProgrammingAssignment1/src/programmingassignment1/Receptionist.java
// COSC 2320 Summer 2015
// Ali Elsaadi
// Progamming Assignment 1
// This is my own work; I will not post
package programmingassignment1;


public class Receptionist extends Adminstrator {
    private boolean answering;
    public Receptionist ()
    {
        super (" ", 0, " " );
        answering = false;
    }
    public Receptionist (String empName, int empNum, String dept, boolean ans)
    {
        super (empName, empNum, dept);
        answering = ans;
    }
    public void setIsAnswering( boolean isA)
    {
        answering = isA;
        
    }
    
    public boolean getIsAnswering()
    {
        return answering;
    }
    public String toString()
    {
        char Ans = 'N';
        if (answering)
            Ans = 'Y';
        return super.toString() + "Answering: "  + Ans;
    }
}

File: DS Assignments/ProgrammingAssignment4/ProgrammingAssignment4/ProgrammingAssignment4/Receptionist.cpp
#include "Receptionist.h"

Receptionist::Receptionist()
{
	Receptionist::Adminstrator(" ", 0, " ");
	answering = false;
}
Receptionist::Receptionist(string empName, int empNumber, string dept, bool ans)
{
	Receptionist::Adminstrator(empName, empNumber, dept);
	answering = ans;
}
void Receptionist::setIsAnswering(bool isA)
{
	answering = isA;
}
bool Receptionist::getIsAnswering()
{
	return answering;
}
string Receptionist:: toString()
{
	char Ans = 'N';
	if (answering)
		Ans = 'Y';
	return Receptionist::Adminstrator::toString() + "Answering: " + Ans;
}